new/rfc-knife-bootstrap-client-config.md

```diff
@@ -0,0 +1,216 @@
+---
+RFC: unassigned
+Author: Jay Mundrawala <jdm@chef.io>
+Author: Bryan McLellan <btm@chef.io>
+Status: Draft
+Type: Standards Track
+
+---
+
+# Knife Boostrap `client.rb` Configuration
+
+Knife bootstrap currently only allows a select few `client.rb` parameters
+to be specified during bootstrap. This RFC is an attempt to allow users
+to be able to specify `client.rb` parameters that are not directly known
+to knife bootstrap.
+
+## Background
+
+The `knife-boostrap` command supports a small subset of possible parameters
+a user can tweak in the `client.rb`. It allows setting `node_name`,
+`verify_api_cert`, `ssl_verify_mode`, `http_proxy`, and `no_proxy`. The
```

Comment:
it does set a few other settings dynamically, it's just not passing an argument straight through. I might just drop the list and emphasize that these options have to be hardcoded into a release (if you don't emphasize that later in the text).